packages/modules/viewer-react/src/components/app-ui/providers/ViewerNavigationToolsProvider.tsx

```diff
@@ -29,122 +29,12 @@ export class ViewerNavigationToolsProvider extends StandardNavigationToolsProvid
-      toolbarOrientation === ToolbarOrientation.Horizontal
-    ) {
-      if (
-        !this.defaultItems ||
-        !this.defaultItems.horizontal ||
-        this.defaultItems.horizontal.rotateView
-      ) {
-        items.push(
-          ToolbarHelper.createToolbarItemFromItemDef(
-            10,
-            CoreTools.rotateViewCommand
-          )
-        );
-      }
-      if (
-        !this.defaultItems ||
-        !this.defaultItems.horizontal ||
-        this.defaultItems.horizontal.panView
-      ) {
-        items.push(
-          ToolbarHelper.createToolbarItemFromItemDef(
-            20,
-            CoreTools.panViewCommand
-          )
-        );
-      }
-      if (
-        !this.defaultItems ||
-        !this.defaultItems.horizontal ||
-        this.defaultItems.horizontal.fitView
-      ) {
-        items.push(
-          ToolbarHelper.createToolbarItemFromItemDef(
-            30,
-            CoreTools.fitViewCommand
-          )
-        );
-      }
-      if (
-        !this.defaultItems ||
-        !this.defaultItems.horizontal ||
-        this.defaultItems.horizontal.windowArea
-      ) {
-        items.push(
-          ToolbarHelper.createToolbarItemFromItemDef(
-            40,
-            CoreTools.windowAreaCommand
-          )
-        );
-      }
-      if (
-        !this.defaultItems ||
-        !this.defaultItems.horizontal ||
-        this.defaultItems.horizontal.viewUndoRedo
-      ) {
-        items.push(
-          ToolbarHelper.createToolbarItemFromItemDef(
-            50,
-            CoreTools.viewUndoCommand
-          )
-        );
-        items.push(
-          ToolbarHelper.createToolbarItemFromItemDef(
-            60,
-            CoreTools.viewRedoCommand
-          )
-        );
-      }
-    } else if (
-      toolbarUsage === ToolbarUsage.ViewNavigation &&
-      toolbarOrientation === ToolbarOrientation.Vertical
-    ) {
-      if (
-        !this.defaultItems ||
-        !this.defaultItems.vertical ||
-        this.defaultItems.vertical.walk
-      ) {
-        items.push(
-          ToolbarHelper.createToolbarItemFromItemDef(
-            10,
-            new ToolItemDef({
-              toolId: "View.LookAndMove",
-              iconSpec: "icon-walk",
-              execute: () =>
-                IModelApp.tools.run(
-                  "View.LookAndMove",
-                  IModelApp.viewManager.selectedView
-                ),
-              labelKey: "iTwinViewer:tools.walkTool",
-            })
-          )
-        );
-      }
-      if (
-        !this.defaultItems ||
-        !this.defaultItems.vertical ||
-        this.defaultItems.vertical.toggleCamera
-      ) {
-        items.push(
-          ToolbarHelper.createToolbarItemFromItemDef(
-            20,
-            CoreTools.toggleCameraViewCommand
-          )
-        );
-      }
-    }
-    return items;
+  // todo replace walk view tool with look and move tool
+  override getToolbarItems(): readonly ToolbarItem[] {
```

Comment:
Missing functionality or the override can be removed entirely if no longer needed.

Reply:
need to circle back to this. the viewer always used the LookAndMove tool as thats what Design Review uses. Wonder if the viewer should just use what AppUI uses or DR. further discussion needed